test/rate-limit-policies.yml

```diff
@@ -0,0 +1,2 @@
+total-certs:
+  threshold: 10000
```

Comment:
How was this number derived? I think we might have close this many in the beta signups. 

I assume our plan to bump this again before GA but not during the beta (which is why I'm thinking this is maybe too low, but am asking to check my assumptions). 

It would be cool to get a ticket somewhere (perhaps not this GH repo's tracker) to bump it for GA, at least.